deploy/crds/smartgateway.infra.watch_v2_smartgateway.metrics_cr.yaml

```diff
@@ -10,11 +10,11 @@ spec:
   bridge:
     enabled: true
     amqpUrl: "amqp://amq-interconnect:5672/collectd"
-    unix_socket_path: "/tmp/smartgateway"
-    socket_block: true
-    ring_buffer_count: 15000
-    stats_period: 60
-    stop_count: 0
+    unixSocketPath: "/tmp"
```

Comment:
The socket path shouldn't be pointing at a directory

Reply:
If I understand it correctly, the "/smartgateway" is appended [here](https://github.com/infrawatch/smart-gateway-operator/blob/7cb7ee51f9b944f557eba121d359a5e50e1c5beb/roles/smartgateway/templates/deployment.yaml.j2#L61) so in the end it points to /tmp/smartgateway

Reply:
ah you're right! My mistake

Reply:
Oops! I got a bit overzealous on the cherry-pick I think...

Need to adjust this, and to stable-1.3...

Reply:
We have the same change already cherry picked to stable-1.3, and it sounds like this should be correct, so I'm going to merge it. If additional changes are required just open a new PR and we'll get everything sorted across both branches. Thanks!